homeassistant/components/test.json

```diff
@@ -0,0 +1,32 @@
+{
+  'homeassistant': OrderedDict
```

Comment:
I see github colours these red. Proper JSON delimiters probably only include double quotes `"`. It's only Python that is very lenient